### PR TITLE
Edits to 10022 (Paizuri Executioner)

### PR DESCRIPTION
**Pull request by HotHeadEmperor**
Line 39 is untranslated

### CODE REVIEW

File: CSV/Language/English/Card/10022.txt [hunk 1]
```diff
@@ -53,37 +53,37 @@ EU22_paizuriAnime,
...
 あと少しでとどめだったのに、あなたのおかげで体力も元通りです❤エナジーごちそうさまです❤くすくす❤,Just a little more and it would have been the finishing blow; but thanks to you, my strength has fully returned.❤ Thank you for the energy meal. *Giggle*❤,
 相手の自滅で勝てるなんて楽な戦いでした❤エアパイズリ、次も見せてあげますよ？だから、また負けてくださいね❤くすくす❤,That was an easy victory; winning because of my opponent masturbating~❤ I'll show you my air paizuri again; so please lose to me next time~❤ *Giggle*❤
 パイズリでたくさん気持ちよくなれてよかったですね❤わたしたちの教えの素晴らしさ、身をもって知ることができたでしょう？くすくす❤,I'm glad that you enjoyed my paizuri that much~❤ You now see how wonderful our teachings are; don't you? *Giggle*❤
 シスターの使う女性モンスターに敗れた僕は、エナジーを奪われ膝をついて動けなくなっていた。,I collapsed to my knees in front of the female monster used by the Sister; the beautiful monster having drained away my energy.      
-ううっ…。,OOogh...
-教えの素晴らしさ、よく分かっていただけましたか？女の体はとっても気持ちのいいものでしょう？,Do you understand how wonderful our teachings are now? How a woman's body willpower feel better than anything else?
+ううっ…。,Ooogh...
+教えの素晴らしさ、よく分かっていただけましたか？女の体はとっても気持ちのいいものでしょう？,Do you understand how wonderful our teachings are now? How a woman's body will feel better than anything else?
```
Comment (Giratena) [inline CSV/Language/English/Card/10022.txt:86]: Maybe remove "will" here entirely -> "How a woman's body feels better than anything else?"

**Review by Giratena — APPROVED** [review]
A very good clean up here! Just one small change then I'll merge ^^

**Review by Giratena — CHANGES_REQUESTED** [review]
Whoops was meant to be change request, sorry

File: CSV/Language/English/Card/10022.txt (hunk 1 above)
Comment (HotHeadEmperor) [reply CSV/Language/English/Card/10022.txt]: Adjustment made.

**Pull request merged**